discretize/curvilinear_mesh.py

```diff
@@ -689,18 +522,6 @@ def boundary_faces(self):
 
     @property
     def boundary_face_outward_normals(self):
-        """Outward normals of boundary faces
-
-        For all boundary faces in the mesh, this property returns
-        the unit vectors denoting the outward normals to the boundary.
-        The returned quantity is a numpy array of shape
-        (n_boundary_faces, dim).
-
-        Returns
-        -------
-        (n_boundary_faces, dim) numpy.ndarray of float
-            Outward normals of boundary faces
-        """
```

Comment:
Are the removed comments somewhere else now?

Reply:
The documentation for the all of the shared functionality lives on `BaseMesh` now, end these functions essentially inherit that documentation. So it's all there still, just consolidated!